### ISSUE TITLE
Add support fo [/] and [%] cookies for TODOs and DONEs

### ISSUE THREAD

**Issue by gerazov** [nvim-orgmode/orgmode#305]
### Does this feature exist in Emacs orgmode core?

N/A

### Feature value

It would be useful to have this for large tasks with subtasks that can be scheduled in time.

It's related to #287 in which support for nested list items was added.

**Comment by mcepl** [comment]
Yes, it does exist in the Emacs orgmode, that’s how orgmode works. E.g., this should work:
```org
** TODO my task [/]
*** TODO which has a subtask
```

**Comment by gerazov** [comment]
Awesome 🤟
